site/blog/unlisted/iasql-on-ec2.mdx

```diff
@@ -208,3 +201,167 @@ WHERE id = (SELECT id
+```sql
+SELECT iasql_install('aws_route53');
+```
+Alright, now let's `INSERT` a row into the `hosted_zone` table to direct the IaSQL engine to create a new Route53 hosted zone for us.
+
+:::note
+To know which tables are supported by the `aws_route53` module, you can visit [`aws_route53` module documentation](https://iasql.com/docs/modules/aws/aws_route53/).
+:::
+
+```sql
+SELECT iasql_begin();
+
+INSERT INTO hosted_zone (domain_name)
+VALUES ('<your-domain>');
+
+SELECT iasql_commit();
+```
+
+:::note
+Simple `INSERT`, right? But wait! What are those `iasql_begin` and `iasql_commit` functions called before and after our insert command?
+
+Quick answer is that if you need your changes to be applied now instead of eventually in the future, you'll wrap the changes between an `iasql_begin` and an `iasql_commit`. For more info on what they do, visit [our documentation on IaSQL transactions](https://iasql.com/docs/transaction/).
+:::
+
+Now that we have created our hosted zone, we need to set our domain's nameservers to the ones that Route53 provides us. For each public hosted zone that you create, Amazon Route53 [automatically creates](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/SOA-NSrecords.html) a name server (NS) record and a start of authority (SOA) record. So we need to log into our domain registrar and set the name server values to the four name server values that Route53 has set. How this is done exactly varies from registrar to registrar, but we can get those NS values with a `SELECT` query:
+```sql
+SELECT record as name_server
+FROM resource_record_set
+WHERE parent_hosted_zone_id = (SELECT id FROM hosted_zone WHERE domain_name = '<your-domain>')
+  AND record_type = 'NS';
+```
+After updating the name servers in the domain settings, we need to wait for those values to be propagated. This can happen in minutes, or even up to 48 hours depending on the registrar.
+
+
+:::note
+You can find out which NS records are accessible from your DNS with the [`dig` command](https://linux.die.net/man/1/dig):
+
+```sh
+dig <your-domain> ns
+```
+
+It should look something like this when ready:
+
+```sh
+$ dig skybase.dev ns
+
+; <<>> DiG 9.18.1-1ubuntu1.3-Ubuntu <<>> skybase.dev ns
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 27469
+;; flags: qr rd ra; QUERY: 1, ANSWER: 4, AUTHORITY: 0, ADDITIONAL: 1
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 65494
+;; QUESTION SECTION:
+;skybase.dev.		                IN	NS
+
+;; ANSWER SECTION:
+skybase.dev.	        21600	IN	NS	ns-1243.awsdns-27.org.
+skybase.dev.	        21600	IN	NS	ns-1944.awsdns-51.co.uk.
+skybase.dev.	        21600	IN	NS	ns-49.awsdns-06.com.
+skybase.dev.	        21600	IN	NS	ns-697.awsdns-23.net.
+
+;; Query time: 60 msec
+;; SERVER: 127.0.0.53#53(127.0.0.53) (UDP)
+;; WHEN: Wed Mar 29 20:48:04 CDT 2023
+;; MSG SIZE  rcvd: 179
+```
+:::
+
+#### Issue An SSL Certificate For Our Domain
```

Comment:
what does this look like? the fact that this grayed out makes me think if might not properly parsing the sections

Reply:
Docusaurus' MDX markdown is *not* Github-style markdown, so I do highly recommend actually pulling the branch, and running:

```
cd site
yarn
yarn start:local
```

and navigating to the blogpost to see what it'll actually look like. That's what I've been doing on this PRs.

Reply:
<img width="786" alt="image" src="https://user-images.githubusercontent.com/5357435/229236517-eebd203c-a637-462d-98b4-035159601002.png">